### PR TITLE
KH-116: HC Screening Form Adjustments

### PR DESCRIPTION
**Pull request by kdaud**
Refer to [KH-116](https://issues.openmrs.org/browse/KH-116)

### CODE REVIEW

**Comment by kdaud** [comment]
UX

[ScreeningForm.webm](https://user-images.githubusercontent.com/38831673/231692626-2d4d55c0-c978-4b06-b863-8a85d8e92af0.webm)

File: configs/openmrs_config/concepts/lifestyle.csv [hunk 1]
```diff
@@ -9,6 +9,9 @@ d7035879-6668-4cb0-87ab-a1311ccb774c,,,Counselled on tobacco cessation,បាន
 dc833bb7-2d04-4be0-ab58-47b9fb2bf771,,,Counselled on diet & physical activity,បានផ្តល់​ប្រឹក្សា​អំពី​របបអាហារ និងកាយវប្បកម្ម (សកម្មភាពរាងកាយ),Counselled on diet & physical activity,បានផ្តល់​ប្រឹក្សា​អំពី​របបអាហារ និងកាយវប្បកម្ម (សកម្មភាពរាងកាយ),,,Question,Coded,Yes;No,,,
 805c3a0b-cd38-4ed6-b4f8-f3af0fc118ad,,,Started on medicine,បានចាប់ផ្តើមប្រើឱសថ,Started on medicine,បានចាប់ផ្តើមប្រើឱសថ,,,Question,Coded,Yes;No,,,
 62e5ac97-0fb6-478b-bd30-2157374b3cc0,,,Medication,ឱសថ,Medication,ឱសថ,,,Question,Coded,Metformin;Glibenclamide+Gliclazide;Insulin (Long-acting);Insulin (Rapid acting);Insulin (Mixed);Simvastatin;Fenofibrate;Aspirin;Amlodipine;Atenolol;Enalapril;Hydralazine;Hydrochlorothiazide;Methyldopa;Nicardipine;Nifedipine;Losartan;Captopril,,,
+0eb2b443-57d6-4998-8596-c9303d33abf0,,,Default,,Default,,,,Misc,N/A,,,,
+c84f1b1c-9f19-4999-a618-0c256d132365,,,Deceased,,Deceased,,,,Misc,N/A,,,,
+8ee37ce5-cb9d-44a6-8adc-6efb39b07b75,,,Discharged service,,,,,,Question,Coded,Default;Deceased,,,
```
Comment (rbuisson) [inline configs/openmrs_config/concepts/lifestyle.csv:14]: Please rename to better name:
`Reason for discharge` seems more decent.

File: configs/openmrs_config/concepts/management.csv [hunk 2]
```diff
@@ -3,4 +3,7 @@ Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:k
 a085b404-2596-48d6-a454-cd164d4a83fc,,,Monitor every 6 months (20% to <30%),ធ្វើការតាមដានរៀងរាល់៦ខែម្តង (ហានិភ័យ 20% ទៅ <30%),Monitor every 6 months (20% to <30%),ធ្វើការតាមដានរៀងរាល់៦ខែម្តង (ហានិភ័យ 20% ទៅ <30%),,,Misc,N/A,,,,
 9b305de5-9df6-4db9-820d-9a24541418ab,,,Monitor every 3 months (>=30%),ធ្វើការតាមដានរៀងរាល់៣ខែម្តង (ហានិភ័យ >=30%),Monitor every 3 months (>=30%),ធ្វើការតាមដានរៀងរាល់៣ខែម្តង (ហានិភ័យ >=30%),,,Misc,N/A,,,,
 7a67e555-1320-4197-a5df-54be3e68a1fe,,,Follow-up screening CVD risk factors,តាមដានពិនិត្យរុករកកត្តាហានិភ័យជំងឺសរសៃឈាម-បេះដូង,Follow-up screening CVD risk factors,តាមដានពិនិត្យរុករកកត្តាហានិភ័យជំងឺសរសៃឈាម-បេះដូង,,,Question,Coded,Monitor every year (<20%);Monitor every 6 months (20% to <30%);Monitor every 3 months (>=30%),,,
+4d32413a-95a4-45bf-95a0-ff81e26cfd35,,,CVD Risk Factors,,CVD Risk Factors,,,,Misc,N/A,,,,
+33f058fd-8341-4ca1-b843-be71e5fe59ac,,,Diabetes mellitus/Hypertension artérielle,,DM / HTA,,,,Misc,N/A,,,,
```
Comment (rbuisson) [inline configs/openmrs_config/concepts/management.csv:7]: I'm surprised of the "Hypertension artérielle" written in French like that. Where does that come from?

Comment (rbuisson) [reply configs/openmrs_config/concepts/management.csv:7]: `DM / HTA` -> `DM/HTA`

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: configs/openmrs_config/concepts/management.csv (hunk 2 above)
Comment (kdaud) [reply configs/openmrs_config/concepts/management.csv:7]: @rbuisson, could you let me know the right spelling for the full name of `HTA` ?

Comment (kazlaw) [reply configs/openmrs_config/concepts/management.csv:7]: @kdaud if it is Hypertension the translation is in this Google sheet [here](https://docs.google.com/spreadsheets/d/13PHkzorfTmYRjuIJv3fCml3zBAuDba6Mgdpu4H7cAg8/edit#gid=0&range=68:68)

**Comment by kdaud** [comment]
@rbuisson we can have this PR merged though there still pends some khmer translation that Pengby hasn't provided yet. When he does, I will make a follow up commit.

**Review by rbuisson — APPROVED** [review]

**Pull request merged**